app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -73,6 +75,8 @@ class SettingsRobot {
     // PRIVACY SECTION
     fun verifyPrivacyHeading() = assertPrivacyHeading()
 
+    fun verifyHTTPSOnlyModeButton() = assertHTTPSOnlyModeButton()
```

Comment:
Probably clearer if we rename this function to `verifyHTTPSOnlyModeToggle` - it's a toggle right?

Reply:
No, it's not a toggle, it's a button in "Settings"
The other items have the same naming.
<img width="384" alt="image" src="https://user-images.githubusercontent.com/51314259/162950871-d5d50ebb-dba3-4736-a731-34548c1fb62e.png">